app/src/main/java/de/danoeh/antennapod/preferences/PreferenceUpgrader.java

```diff
@@ -34,6 +34,10 @@ public static void checkUpgrades(Context context) {
 
     private static void upgrade(int oldVersion) {
         if (oldVersion == -1) {
+            //New installation
+            if(UserPreferences.getUsageCountingMillis() < 0){
```

Comment:
Why is this "if" needed? I think the code should only be called on the very first install. We therefore know that the usage stats are not yet counting. Did I miss something?

Reply:
Yes this is not necessary, but just to be sure ;)